src/test/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployerTests.java

```diff
@@ -222,6 +230,25 @@ public void deployWithDeploymentServiceAccountNameDeploymentPropertyOverride() {
 		assertThat(podSpec.getServiceAccountName().equals("overridesan"));
 	}
 
+	@Test
+	public void deployWithGlobalTolerations() {
```

Comment:
thanks for adding this one -- i think as mentioned above we should also have a test for overrides. for example, if in the server config there is a toleration defined globally, then a toleration configured on a specific app deployment, the deployment property defined toleration should take precedence

Reply:
I have added the override one as well in the lastest commit.

Reply:
thx